### PR TITLE
first couple of steps in oxd trade

### PR DESCRIPTION
**Pull request by jorijnsmit**
steps and details in #333 

last func still has to be written (the actual oxd swap on fantom)

### CODE REVIEW

File: scripts/issue/333/swap_dfd_comp_for_oxd.py [hunk 1]
```diff
@@ -0,0 +1,66 @@
...
+    VAULT.take_snapshot([DFD])
+    TROPS.take_snapshot([DFD])
+
+    # since dfd isnt owned by trops yet, overwrite from param
+    DFD.transfer(TROPS, DFD.balanceOf(VAULT), {'from': VAULT.account})
+
+    VAULT.print_snapshot()
+    TROPS.print_snapshot()
+
+    TROPS.post_safe_tx(call_trace=True)
```
Comment (petrovska-petro) [inline scripts/issue/333/swap_dfd_comp_for_oxd.py:27]: this needs to be post from `VAULT`

File: scripts/issue/333/swap_dfd_comp_for_oxd.py [hunk 2]
```diff
@@ -0,0 +1,66 @@
...
+    COMP.approve(TROPS.cow.vault_relayer, COMP.balanceOf(TROPS))
+    TROPS.post_safe_tx(call_trace=True)
+
+
+def swap_for_usdc():
+    # swap dsd for usdc
+    # swap comp for usdc
+    TROPS.init_cow()
+    TROPS.cow.market_sell(DFD, USDC, DFD.balanceOf(TROPS), coef=.985)
+    TROPS.cow.market_sell(COMP, USDC, DFD.balanceOf(TROPS), coef=.985)
```
Comment (petrovska-petro) [inline scripts/issue/333/swap_dfd_comp_for_oxd.py:42]: here should be `COMP.balanceOf(TROPS)`, DFD is duplicated

**Comment by jorijnsmit** [comment]
thanks @petrovska-petro , i resolved the issues you found in a487282.

File: scripts/issue/333/swap_dfd_comp_for_oxd.py [hunk 3]
```diff
@@ -0,0 +1,66 @@
...
+    TROPS.init_anyswap()
+    info = TROPS.anyswap.get_token_bridge_info(
+        registry.eth.treasury_tokens.USDC, chain_id_origin=1
+    )
+    anytoken = info['anyToken']['address']
+
+    router = interface.IAnyswapV6Router(info['router'], owner=TROPS.account)
+    USDC.approve(router, mantissa)
+    router.anySwapOutUnderlying['address,address,uint256,uint256'](
+        anytoken, TROPS, mantissa, 250
```
Comment (petrovska-petro) [inline scripts/issue/333/swap_dfd_comp_for_oxd.py:58]: destination in the other side, should be perhaps the [FTM_TROPS](https://github.com/Badger-Finance/badger-multisig/blob/main/helpers/addresses.py#L696), sending to the `eth` trops will be to lose the funds

**Review by petrovska-petro — APPROVED** [review]

**Review by jalbrekt85 — APPROVED** [review]
lgtm

**Pull request merged**